src/platform/brand-consolidation/va-subdomain.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Returns whether the current hostname is a VA subdomain
+ * @returns {boolean}
+ * @module platform/brand-consolidation/is-subdomain
```

Comment:
tiny typo: `is-subdomain` -> `va-subdomain`

Reply:
Thanks, fixed.